kt/godot-library/src/main/kotlin/godot/core/memory/TransferContext.kt

```diff
@@ -52,9 +52,9 @@ internal object TransferContext {
         return ret
     }
 
-    fun callMethod(ptr: VoidPtr, methodIndex: Int, expectedReturnType: VariantType) {
+    fun callMethod(ptr: VoidPtr?, methodIndex: Int, expectedReturnType: VariantType) {
```

Comment:
I would instead make the called directly input 0 here instead of adding a nullable + test.